### PR TITLE
replace acquireTokenSilent to ssoSilent method

### PR DESCRIPTION
**Pull request by dandresini**
If I edit the information from msal EditProfile flow and then I verify acquire token information the method not refresh the information if i use acquireTokenSilent.

### CODE REVIEW

File: samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx [hunk 1]
```diff
@@ -22,9 +22,10 @@ const ProtectedContent = () => {
         if (!atsResponse && account && inProgress === InteractionStatus.None) {
             const request = {
                 ...loginRequest,
-                account: account
+                loginHint: account.username
             };
-            instance.acquireTokenSilent(request).then((response) => {
+
+            instance.ssoSilent(request).then((response) => {
```
Comment (jasonnutter) [inline samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx]: I don't think we want to invoke ssoSilent this often. Have you tried invoking `acquireTokenSilent` with `forceRefresh` set to `true`?

Comment (dandresini) [reply samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx]: I'll test with force refresh and I'll send a report of result.

Comment (dandresini) [reply samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx]: I use the button "Edit profile" that call the "_instance.loginPopup(b2cPolicies.authorities.editProfile)_" to change the city from **Bari** to **Torino** as image below (in the claims i pass the city information)
B2c_EditProfile

When I show the information of token, the city claim isn't the last changed (as you said I use _forceRefresh: true_)

result_request

Comment (tnorling) [reply samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx]: B2C will keep returning the same token claims when the RT is used to acquire it so ssoSilent would be the correct way to refresh it silently. However, I agree with @jasonnutter, we shouldn't call ssoSilent every time, only when something has been updated (to ensure the cache has the updated token, and then use acquireTokenSilent for all subsequent requests).

Comment (dandresini) [reply samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx]: I used in editprofile component the method "loginRedirect" and all work fine. With loginPopUp ther'is a problem in refresh token

**Comment by tnorling** [comment]
Demonstration of the edit profile flow will be added in #5015 - closing in favor of that PR

**Pull request closed without merging**